.github/workflows/python-app.yml

```diff
@@ -24,16 +24,13 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements/requirements.txt') }}
```

Comment:
```suggestion
          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements/*.txt') }}
```

Reply:
I think we shouldn't do this here because we don't want to install the linting packages in testing.